### ISSUE TITLE
Rule proposal: the `never` option for the `numeric-separators-style`

### ISSUE THREAD

**Issue by vanyauhalin** [sindresorhus/eslint-plugin-unicorn#2493]
### Description

I would appreciate a flag or option to disallow the use of a separator for numbers.

### Fail

```js
const n = 10_000
```

### Pass

```js
const n = 10000
```

### Proposed rule name

numeric-separators-style

### Additional Info

To avoid adding a new flag it could be achieved using a negative (exact `-1`) value:

```js
{
  "unicorn/number-literal-case": ["error", -1],
}
```

```js
{
  "unicorn/number-literal-case": {
    hexadecimal: -1,
    binary: -1,
    octal: -1,
    number: -1,
  },
}
```

```js
{
  "unicorn/number-literal-case": {
    hexadecimal: {minimumDigits: -1},
    binary: {minimumDigits: -1},
    octal: {minimumDigits: -1},
    number: {minimumDigits: -1},
  },
}
```

**Comment by fregante** [comment]
Duplicate of https://github.com/sindresorhus/eslint-plugin-unicorn/issues/868

**Comment by vanyauhalin** [comment]
I am going to close this issue. I see that this idea is probably against the views of the maintainers of this rule set. That is fine; I do not want to force it. Moreover, I have found the [es-x/no-numeric-separators](https://eslint-community.github.io/eslint-plugin-es-x/rules/no-numeric-separators.html) rule that suits me.
